internal/proto/codec/builtin.go

```diff
@@ -947,6 +947,16 @@ func DecodeNullableForSQLQueryId(it *proto.ForwardFrameIterator) *isql.QueryID {
 	return DecodeSqlQueryId(it)
 }
 
+func DecodeNullableForSQLHazelcastJSON(it *proto.ForwardFrameIterator) []driver.Value {
+	if CodecUtil.NextFrameIsNullFrame(it) {
+		return nil
+	}
+	vs, _ := DecodeListMultiFrameContainsNullable(it, func(it *proto.ForwardFrameIterator) (driver.Value, error) {
```

Comment:
Would it make sense to add a comment above this line about why the error handling is ignored?

Reply:
Sure, added at https://github.com/hazelcast/hazelcast-go-client/pull/726/commits/972309f6841a6c712338f131d67636935ede4025